index.html

```diff
@@ -113,9 +120,16 @@ <h2>Lorem ipsum dolor sit</h2>
                     <h2>Lorem ipsum dolor sit</h2>
                     <p>amet consetetur sadipscing elitr</p>
                   </div>
-                  <button class="btn-zoom">
-                    <img src="assets/img/magnifying-glass.png" alt="" />
-                  </button>
+                  <a
+                    data-fslightbox="lightbox-1"
+                    href="	https://ivanakos.github.io/IvanaKos.github.io-Variety/assets/img/gallery-img-2.jpg"
```

Comment:
```suggestion
                    href="https://ivanakos.github.io/IvanaKos.github.io-Variety/assets/img/gallery-img-2.jpg"
```

Reply:
Not needed since I changed the path to relative.